src/authentication/processAuth.ts

```diff
@@ -32,17 +32,19 @@ export class ProcessAuth extends AlfrescoApiClient {
         'basicAuth': { ticket: '' }, type: 'activiti'
     };
 
-    defaultHeaders = {
-        'user-agent': 'superagent'
-    };
-
     constructor(config: AlfrescoApiConfig) {
         super();
         this.storage = new Storage();
         this.storage.setDomainPrefix(config.domainPrefix);
 
         this.className = 'ProcessAuth';
 
+        if (this.isNodeEnv()) {
+            this.defaultHeaders = {
```

Comment:
do we really need custom user agent in any of the cases?

Reply:
yep for some reason login fails